examples/data-objects/inventory-app/src/inventoryList.ts

```diff
@@ -4,21 +4,42 @@
  */
 
 import { DataObjectFactory } from "@fluidframework/aqueduct";
+import { ContainerSchema } from "fluid-framework";
 
-import { TreeDataObject, factory } from "./reactSharedTreeView.js";
+import { TreeDataObject, factory, treeDataObject } from "./reactSharedTreeView.js";
 import { type Inventory, treeConfiguration } from "./schema.js";
 
 /**
- * @internal
+ * For use with high level service-client container API's like AzureClient and OdspClient.
+ * @privateRemarks
+ * TODO: There is no service implementation agnostic client abstraction that can be referred to here (ex: shared by AzureClient and OdspClient).
+ * This makes documenting compatibility with that implicit common API difficult.
+ * It also makes writing service agnostic code at that abstraction level harder.
+ * This should be fixed.
+ *
+ * TODO:
+ * Writing an app at this abstraction level currently requires a lot of boilerplate which also requires extra dependencies.
+ * Since `@fluid-example/example-utils` doesn't provide that boilerplate and neither do the public packages, there isn't a concise way to actually use this container in this example.
+ * This should be fixed.
+ *
+ * TODO:
+ * The commonly used boilerplate for setting up a ContainerSchema based application configures the dev-tools, which would be great to include in this example,
+ * but can't be included due to dependency layering issues.
  */
+export const containerSchema = {
```

Comment:
Where does this get used?  Am I missing it?

Reply:
it is not currently used since using it is totally impractical due to needing a ton of boilerplate and deps. However this is still an example of how we could make tree dataobjects compatible with container schema, which I think is important

Reply:
If the above TODO's were addressed, we could use this here.

Reply:
I have moved this code into a unit test with different docs to clarify.